### DISCUSSION TITLE
Workspace Question (can they not be per screen?)

### DISCUSSION THREAD

**Discussion by ghost** [awesomeWM/awesome#3901] [Q&A]
I've searched the docs and configs and can't find anything on the workspaces. A bug report I found hints that "workspaces" are or are tied to "tags" in the Awesome nomenclature. This is to say forgive me if I've missed the obvious as my searches haven't helped but perhaps my terms are wrong to being new to Awesome and Tiling in general.

So the question is simply can I have one workspace per set of screens? Each XScreen runs 3 monitors and I'd want all 3 monitors to have the same workspace rather than separate. Several Applications I run I run on all screens like my DAW or Blender. In my tests switching workspaces has some odd behaviors where an application is linked to the workspace of the left most screen it's on but I can end up with bits of a single application all over the place.

As a secondary possibility is it possible to trigger all workspaces with the same keystrokes? So if I switch from 3 to 1 have it affect all so everything stays in sync?

**Comment by actionless** [comment]
> As a secondary possibility is it possible to trigger all workspaces with the same keystrokes? So if I switch from 3 to 1 have it affect all so everything stays in sync?

yes, just iterate over each screen in the function of that hotkey instead switching it on the active screen only
